src/Microsoft.AspNetCore.Http.Extensions/UriHelper.cs

```diff
@@ -101,7 +101,7 @@ public static string Encode(Uri uri)
         /// <returns></returns>
         public static string GetEncodedUrl(this HttpRequest request)
```

Comment:
Encode could be an extension, but I think you'd loose context of what you were encoding it for. It's only used here:
https://github.com/aspnet/HttpAbstractions/blob/dev/src/Microsoft.AspNetCore.Http.Extensions/ResponseHeaders.cs#L145